test/integration/urdf_gazebo_extensions.cc

```diff
@@ -99,4 +99,163 @@ TEST(SDFParser, UrdfGazeboExtensionURDFTest)
       EXPECT_TRUE(physics->Get<bool>("implicit_spring_damper"));
     }
   }
+
+  sdf::ElementPtr link0;
+  for (sdf::ElementPtr link = model->GetElement("link"); link;
+       link = link->GetNextElement("link"))
+  {
+    const auto& linkName = link->Get<std::string>("name");
+    if (linkName == "link0")
+    {
+      link0 = link;
+      break;
+    }
+  }
+  ASSERT_TRUE(link0);
+
+  bool foundSensorNoPose {false};
+  bool foundSensorPose {false};
+  bool foundSensorPoseRelative {false};
+  bool foundSensorPoseTwoLevel {false};
+  bool foundIssue378Sensor {false};
+  bool foundIssue67Sensor {false};
+
+  for (sdf::ElementPtr sensor = link0->GetElement("sensor"); sensor;
+       sensor = sensor->GetNextElement("sensor"))
+  {
+    const auto& sensorName = sensor->Get<std::string>("name");
+    if (sensorName == "sensorNoPose")
+    {
+      foundSensorNoPose = true;
+      ASSERT_TRUE(sensor->HasElement("pose"));
+      const auto poseElem = sensor->GetElement("pose");
+
+      const auto& posePair = poseElem->Get<ignition::math::Pose3d>(
+        "", ignition::math::Pose3d::Zero);
+      ASSERT_TRUE(posePair.second);
+
+      const auto& pose = posePair.first;
+
+      EXPECT_FLOAT_EQ(pose.X(), 333.0);
```

Comment:
Can you change all of the `EXPECT_FLOAT_EQ` to `EXPECT_DOUBLE_EQ`. It will make windows happy, see https://build.osrfoundation.org/job/sdformat-ci-pr_any-windows7-amd64/2062/msbuild/new/category.63474753/

Reply:
Okay, I fixed all the equality tests.